process_manager/manager.py

```diff
@@ -0,0 +1,32 @@
+from runner import StandardRunner
+from multiprocessing import Process
+from task import SchedulerTask
+
+
+class ProcessManager:
+    """
+    Main handler for each runner, which is responsible for scheduling the task. 
+    """
+    def __init__(self, timeout: int, size: int):
+        self.realtime_runner = StandardRunner(1, timeout)
+        self.standard_runner = StandardRunner(size, timeout)
+
+    def schedule_with_runner(self, task: SchedulerTask, mode: str):
+        """
+        Schedule a task with the corresponding runner for the given mode.
+        """
+        # TODO: Probably good for enums but right now the original input is a string
+        # so it seems unnecessary to use enums?
+        if mode == 'realtime':
+            return self.realtime_runner.schedule(Process(target=task.target), task.timeout)
+        elif mode == 'standard':
```

Comment:
Not sure why using `str` here instead of `Enum`. We should probably also be raising a `ValueError` if values don't match, but converting from `str` to `Enum` will likely handle this.